controllers/v1/user-extension.js

```diff
@@ -98,6 +98,120 @@ module.exports = class UserExtension extends Abstract {
     })
   }
 
+  /**
+  * @api {get} /kendra/api/v1/user-extension/update/{{userId}} Update user profile
+  * @apiVersion 1.0.0
+  * @apiName Update user profile
+  * @apiGroup User Extension
+  * @apiHeader {String} X-authenticated-user-token Authenticity token
+  * @apiSampleRequest /kendra/api/v1/user-extension/update/e97b5582-471c-4649-8401-3cc4249359bb
+  * @apiParamExample {json} Request-Body:
```

Comment:
@aman-tunerlabs - Who will be calling this API, what is expected to be updated here?

Reply:
@aks30 integration service calling this api